src/__tests__/issue.test.ts

```diff
@@ -244,3 +244,21 @@ test('similarStrings() returns false, when provided two completely different str
 
   expect(winningArea).toStrictEqual(false);
 });
+
+test('areaIsKeyword parameter uses area as keyword', () => {
+  process.env.INPUT_PARAMETERS = JSON.stringify([{
+    area: '@aws-cdk/aws-cognito',
+    keywords: ['nokeywords'],
+    labels: ["a", "b", "c"],
+    assignees: ["d", "e", "f"],
+  }]);
+
+  process.env['INPUT_AREA-IS-KEYWORD'] = 'true';
+
+  const content = ['(@aws-cdk/aws-cognito): This is a title', '(This) is a body'];
+
+  const issue = new Issue(content);
+  const area = issue.determineArea();
```

Comment:
```suggestion
  process.env['INPUT_AREA-IS-KEYWORD'] = 'true';
  const content = ['(@aws-cdk/aws-cognito): This is a title', '(This) is a body'];
  const issue = new Issue(content);
  
  const area = issue.determineArea();
```

In tests it's usually nice to follow a structure like [given, when, then] or [arrange, act, assert] to make it quicker to understand the test. With the above suggestion the test would look like this (excluding my comments):

```ts
test('areaIsKeyword parameter uses area as keyword', () => {
  // Arrange
  process.env.INPUT_PARAMETERS = JSON.stringify([{
    area: '@aws-cdk/aws-cognito',
    keywords: ['nokeywords'],
    labels: ["a", "b", "c"],
    assignees: ["d", "e", "f"],
  }]);
  process.env['INPUT_AREA-IS-KEYWORD'] = 'true';
  const content = ['(@aws-cdk/aws-cognito): This is a title', '(This) is a body'];
  const issue = new Issue(content);

  // Act
  const area = issue.determineArea();

  // Assert
  expect(area).toStrictEqual('@aws-cdk/aws-cognito');
});
```

Here's [my favorite doc on testing best practices](https://github.com/goldbergyoni/javascript-testing-best-practices#-%EF%B8%8F-12-structure-tests-by-the-aaa-pattern) if you're interested.

Reply:
This makes sense, I've seen CDK tests are usually formatted this way!